cmd/thanos/compact.go

```diff
@@ -823,9 +848,9 @@ func (cc *compactConfig) registerFlag(cmd extkingpin.FlagClause) {
 		Default("").EnumVar(&cc.hashFunc, "SHA256", "")
 
 	cc.filterConf = &store.FilterConfig{}
-	cmd.Flag("min-time", "Start of time range limit to compact. Thanos Compactor will compact only blocks, which happened later than this value. Option can be a constant time in RFC3339 format or time duration relative to current time, such as -1d or 2h45m. Valid duration units are ms, s, m, h, d, w, y.").
+	cmd.Flag("min-time", "Start of time range limit to compact. Thanos Compactor will only compact blocks whose max time is at a date more recent than this value. Option can be a constant time in RFC3339 format or time duration relative to current time, such as -1d or -2h45m. Durations must be negative, and valid units are ms, s, m, h, d, w, y.").
```

Comment:
```suggestion
	cmd.Flag("min-time", "Start of time range limit to compact. Thanos Compactor will compact only blocks with a maximum time more recent than this specified value. Option can be a constant time in RFC3339 format or time duration relative to current time, such as -1d or -2h45m. Durations must be negative, and valid units are ms, s, m, h, d, w, y.").
```

Whose is an unnatural possessive for an inanimate object (a block).